tests/selenium/sequential-form-test.js

```diff
@@ -46,13 +45,12 @@ module.exports = function(driver, By) {
       return driver.getCurrentUrl().then(function(url) {
         return url.indexOf('http://localhost:3000/de/thank-you/') === 0;
       });
-    }).then(function() {
```

Comment:
This doesn't actually need to be in a then.

Webdriver already links it's wait functions. We just need to be sure the done bit is inside a then (as the done is not a webdriver thing and thus does not wait.).